packages/kbn-text-based-editor/src/helpers.ts

```diff
@@ -193,6 +193,20 @@ export const getIndicesList = async (dataViews: DataViewsPublicPluginStart) => {
   return indices.map((index) => ({ name: index.name, hidden: index.name.startsWith('.') }));
 };
 
+export const getRemoteIndicesList = async (dataViews: DataViewsPublicPluginStart) => {
+  const indices = await dataViews.getIndices({
+    showAllIndices: false,
+    pattern: '*:*',
+    isRollupIndex: () => false,
+  });
+  const finalIndicesList = indices.filter((source) => {
+    const [_, index] = source.name.split(':');
+    return !index.startsWith('.') && !Boolean(source.item.indices);
+  });
+
+  return finalIndicesList.map((source) => ({ name: source.name, hidden: false }));
+};
```

Comment:
Any reason to make this a separate function?

Reply:
I like it more this way tbh. But is just a personal preference, nothing more